### PR TITLE
Added spacy-wrap to universe

### PR DESCRIPTION
**Pull request by KennethEnevoldsen**
Added spacy-wrap to universe a small package for wrapping fine-tuned huggingface transformers to a spacy pipeline following the same API as spacy-transformers. (Currently limited to classification models)

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [ ] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: website/meta/universe.json [hunk 1]
```diff
@@ -978,6 +978,48 @@
...
+            "slogan": "For Wrapping fine-tuned transformers in spaCy pipelines",
+            "description": "spaCy-wrap is a wrapper library for spaCy for including fine-tuned transformers from Huggingface in your spaCy pipeline allowing inclusion of existing models within existing workflows.",
+            "github": "https://github.com/kennethenevoldsen/spacy-wrap",
+            "pip": "spacy_wrap",
+            "code_example": [
+                "import spacy",
+                "import spacy_wrap",
+                "",
+                "nlp = spacy.blank('en')",
+                "config = {",,
```
Comment (adrianeboyd) [inline website/meta/universe.json:993]:
```suggestion
                "config = {",
```

File: website/meta/universe.json [hunk 2]
```diff
@@ -978,6 +978,48 @@
...
+                "transformer.model.initialize()",
+                "",
+                "doc = nlp('spaCy is a wonderful tool')",
+                "",
+                "print(doc._.clf_trf_data)",
+                "# TransformerData(wordpieces=...",
+                "print(doc._.sentiment)",
+                "# 'positive'",
+                "print(doc._.sentiment_prob)",
+                "# {'prob': array([0.004, 0.028, 0.969], dtype=float32), 'labels': ['negative', 'neutral', 'positive']}",
```
Comment (adrianeboyd) [inline website/meta/universe.json:1012]:
```suggestion
                "# {'prob': array([0.004, 0.028, 0.969], dtype=float32), 'labels': ['negative', 'neutral', 'positive']}"
```

File: website/meta/universe.json [hunk 3]
```diff
@@ -978,6 +978,48 @@
             "category": ["pipeline"],
             "tags": ["pipeline", "danish"]
         },
+        {
+            "id": "spacy-wrap",
+            "title": "spaCy-wrap",
+            "slogan": "For Wrapping fine-tuned transformers in spaCy pipelines",
+            "description": "spaCy-wrap is a wrapper library for spaCy for including fine-tuned transformers from Huggingface in your spaCy pipeline allowing inclusion of existing models within existing workflows.",
+            "github": "https://github.com/kennethenevoldsen/spacy-wrap",
```
Comment (adrianeboyd) [inline website/meta/universe.json:986]:
```suggestion
            "github": "kennethenevoldsen/spacy-wrap",
```

File: website/meta/universe.json [hunk 4]
```diff
@@ -978,6 +978,48 @@
...
+                "# TransformerData(wordpieces=...",
+                "print(doc._.sentiment)",
+                "# 'positive'",
+                "print(doc._.sentiment_prob)",
+                "# {'prob': array([0.004, 0.028, 0.969], dtype=float32), 'labels': ['negative', 'neutral', 'positive']}"
+            ],
+            "thumb": "https://raw.githubusercontent.com/KennethEnevoldsen/spacy-wrap/main/docs/_static/icon.png",
+            "author": "Kenneth Enevoldsen",
+            "author_links": {
+                "github": "https://github.com/KennethEnevoldsen",
```
Comment (adrianeboyd) [inline website/meta/universe.json:1017]:
```suggestion
                "github": "KennethEnevoldsen",
```

**Comment by adrianeboyd** [comment]
This looks very cool!

This is totally fine to include in universe, but a few additional notes if it's helpful:

* It might be helpful for users to make it clear that the component only supports inference and not training, which is a difference from `spacy-transformers`. You can also prevent `spacy train` from trying to update it by setting the property `is_trainable` in the pipeline component to `False`, which might avoid some unexpected behavior or crashes.
* The repo seems to be missing a license and it might be nicer for reference/maintenance if the `spacy-transformers` code was linked a little more explicitly to the original repo.

**Comment by adrianeboyd** [comment]
Let me see if I can turn your comment into a discussion. It might take a few steps, but I'm curious to see exactly what it will do in the background...

**Comment by svlandeg** [comment]
--> Let's continue the discussion over at https://github.com/explosion/spaCy/discussions/10200 😉

**Pull request merged**